### ISSUE TITLE
[FEATURE REQUEST] Rearrange the bottom toolbar to accommodate all the various settings in the space the filters permanently occupy

### ISSUE THREAD

**Issue by thisisartman** [Akylas/OSS-DocumentScanner#335]
**Is your feature request related to a problem? Please describe.**
Currently, the filters permanently occupy the bottom toolbar space, with all the other settings having to adjust into a pop-up menu.

**Describe the solution you'd like**
Shift the filters under a section of their own, thus making that portion of the toolbar space accessible to other settings like the "Brightness & Contrast" , the "Image Enhancement" settings or the OCR settings.

**Describe alternatives you've considered**
Microsoft Lens has the settings tucked away under a button as shown in the attached recording:
https://github.com/user-attachments/assets/e72834f1-78f7-41f4-ad38-27d83c78d870

I'll be the first to admit that there's tons of UX problems with MS Lens. That constantly expanding and contracting bottom toolbar is not ideal. I actually prefer your implementation of a static toolbar that has dynamic elements. But the fact that I can switch between settings is something worth exploring...

**Additional context**

This is how the regular edit page looks.
![Screenshot_20241111-163944_OSS Document Scanner.png](https://github.com/user-attachments/assets/17f619d1-b7c6-4c79-975f-bb6092712a1e)

Here's all the other settings being accessed:
![Screenshot_20241111-163917_OSS Document Scanner.png](https://github.com/user-attachments/assets/bc8b68e8-d8b8-4c95-91f4-dc9399f7b23a)

![Screenshot_20241111-163922_OSS Document Scanner.png](https://github.com/user-attachments/assets/1de4cbac-5fe0-4494-b027-7b0097f8fa9c)

![Screenshot_20241111-163929_OSS Document Scanner.png](https://github.com/user-attachments/assets/b307e9ba-6d23-422b-9efd-d251989f41f4)

**Comment by farfromrefug** [comment]
@thisisartman yes i agree this should be improved. I cant see your video though. Cant open it.

**Comment by thisisartman** [comment]
> I cant see your video though. Cant open it.

Weird.
Let me try adding it here... It's just a recording of the edit page of MS Lens so worst case scenario you could just open it up on your end.
https://github.com/user-attachments/assets/8ecbbb91-9117-4d92-8f04-d1e1b4bb85a4

**Comment by farfromrefug** [comment]
@thisisartman so basically you want me to remove the filter bar, thus sliding all other parameters down, and make the filters a modal opening from a button like other settings?(brightness, enhance...)

**Comment by thisisartman** [comment]
> @thisisartman so basically you want me to remove the filter bar, thus sliding all other parameters down, and make the filters a modal opening from a button like other settings?(brightness, enhance...)

Yes, but don't collapse the bottom area. Have the edit page open with the filters by default or something else would work as well...
